src/3_PB/PianoDiProgetto/sections/ConsuntivoSprint/NonoSprint.typ

```diff
@@ -0,0 +1,20 @@
+#import "../../const.typ": Re_cost, Am_cost, An_cost, Ve_cost, Pr_cost, Pt_cost
+#import "../../functions.typ": rendicontazioneOreAPosteriori, rendicontazioneCostiAPosteriori, glossary
+
+
+== Nono #glossary[sprint]
+
+*Inizio*: Venerdì 16/01/2024
+
+*Fine*: Giovedì 22/01/2024 
+
+#rendicontazioneOreAPosteriori(sprintNumber: "09")
+
+#rendicontazioneCostiAPosteriori(sprintNumber: "09")
+
+=== Analisi a posteriori
+La retrospettiva ha evidenziato come il preventivo del totale delle ore sia stato rispettato. Tuttavia possiamo notare alcune differenze nell'ammontare delle ore di alcuni ruoli. In particolare, il ruolo del Verificatore è quello che risulta più differente rispetto al preventivo, con una totale delle ore inferiore. Questo non ci risulta di particolare preoccupazione. Infatti, si motiva con il fatto che le modifiche e integrazioni apportate ai documenti sono state minori rispetto a #glossary[sprint] passati e avendo due Verificatori la verifica è stata piuttosto rapida. Notiamo invece l'accuratezza del preventivo per il ruolo di Progettista, che nonostante venga utilizzato per la prima volta, con l'inizio della stesura delle _Specifiche Tecniche v1.0_, risulta esser stato stimato con precisione, indice di una maturata consapevolezza nella pianificazione delle attività. Con un leggero sforamento le ore da Analista preventivate hanno permesso di rispettare la pianificazione e di completare la correzione dell'_Analisi dei Requisiti v1.0_ secondo le correzioni ricevute in revisione #glossary[RTB].
+Una strategia efficace per inibire il rischio RC1 è stata la scelta di svolgere gli #glossary("Stand-up meeting") remoti con una frequenza di due volte a settimana, che in unione al meeting interno settimanale, ha permesso di tenere sotto controllo l'avanzamento delle attività e di discutere tempestivamente eventuali problematiche, risultando un impegno sostenibile per il gruppo e quindi una pratica da mantenere anche in futuro.
```

Comment:
specificherei il fatto che comunque si temeva l'affiorazione del rischio in quanto concluse le lezioni ci si è accorti che il team non aveva modo di vedersi in presenza come avveniva durante il primo periodo, anche se è un po una roba così sembra abbastanza pensata

Reply:
Ho provato ad aggiungerlo in collegamento alla parte sugli stand up.

Reply:
okay si bene